content/en/integrations/guide/source-code-integration.md

```diff
@@ -489,20 +492,35 @@ If you're using the GitHub integration, click **Connect to preview** on error fr
 [101]: https://app.datadoghq.com/functions?cloud=aws&entity_view=lambda_functions
 
 {{% /tab %}}
-{{% tab "CI Visibility" %}}
+{{% tab "Test Visibility" %}}
 
-You can see links from failed test runs to their source repository in **CI Visibility**.
+You can see links from failed test runs to their source repository in **Test Visibility**.
 
-1. Navigate to [**CI** > **Test Runs**][101] and select a failed test run.
-2. Scroll down to the **Source Code** section and click the **View Code** button to open the test in its source code repository.
+1. Navigate to [**Software Delivery** > **Test Visibility** > **Test Runs**][101] and select a failed test run.
+2. Click the **View on GitHub** button to open the test in its source code repository.
 
-{{< img src="integrations/guide/source_code_integration/ci-failed-test-blur.png" alt="Link to GitHub from the CI Visibility Explorer" style="width:100%;">}}
+{{< img src="integrations/guide/source_code_integration/test_run_blurred.png" alt="Link to GitHub from the CI Visibility Explorer" style="width:100%;">}}
 
 For more information, see [Enhancing Developer Workflows with Datadog][102].
 
 [101]: https://app.datadoghq.com/ci/test-runs
 [102]: /tests/developer_workflows/#open-tests-in-github-and-your-ide
 
+{{% /tab %}}
+{{% tab "Code Analysis" %}}
+
+You can see links from failed Static Analysis and Software Composition Analysis scans to their source repository in **Code Analysis**.
+
+1. Navigate to [**Software Delivery** > **Code Analysis**][101] and select a repository.
+2. In the **Code Vulnerabilities** or **Code Quality** view, click on a code vulnerability or violation. In the **Details** section, click the **View Code** button to open the flagged code in its source code repository.
+
+{{< img src="integrations/guide/source_code_integration/code-analysis-scan.png" alt="Link to GitHub from the Code Analysis Code Vulnerabilities view" style="width:100%;">}}
+
+For more information, see the [Code Analysis documentation][102].
+
+[101]: https://app.datadoghq.com/ci/code-analysis
+[102]: /code_analysis/#set-up-the-github-integration
```

Comment:
```suggestion
[102]: /code_analysis/
```

minor nit this links "see the code analysis" documentation to the setting up the Github integration, might be better to just link to the top of the page.